vcd/resource_vcd_nsxt_alb_controller.go

```diff
@@ -178,6 +178,9 @@ func setNsxtAlbControllerData(d *schema.ResourceData, albController *types.NsxtA
 	dSet(d, "description", albController.Description)
 	dSet(d, "url", albController.Url)
 	dSet(d, "username", albController.Username)
-	dSet(d, "license_type", albController.LicenseType)
+	// Avoids polluting state for API versions >= v37.0
+	if albController.LicenseType != "" {
```

Comment:
@Didainius just to have your thoughts here for this conditional. The idea is that if LicenseType or SupportedFeatureSet is not coming populated, don't touch the state to leave the field to nil instead of a blank string `""`, but not sure if this is a good practice.

Reply:
This is agreed and changed, but leaving a note.

> If someone had 10.3 and upgraded to 10.4 - their “license_type” field will be stuck with an old value from 10.3
> Because if you don’t do dSet - it will persist the old value.